src/dynamo/models/content.js

```diff
@@ -191,16 +191,23 @@ module.exports = {
       throw new Error('Invalid document: No partition key provided');
     }
 
+    //If the owner is a User object, get its Id without modifying the incoming document
+    let ownerId;
     if (document.owner.id) {
-      document.owner = document.owner.id;
```

Comment:
In the old code after this method finishes the document object was modified, which could cause unexpected failures in a situation like:
```
//in route
const content = Content.getById(...)
//content.owner = User object
content.title = 'blah'
await Content.update(content)
//content.owner = user id string
do something with content.owner.blah fails because its no longer an object
```
In the typescript conversions of dynamo I've been removing this behaviour unintentionally, because its creating a new object for the put action from the original document.